packages/@react-spectrum/s2/stories/CardView.stories.tsx

```diff
@@ -154,10 +155,14 @@ export const Example = (args: CardViewProps<any>, {viewMode}) => {
 Example.args = {
   loadingState: 'idle',
   onAction: null,
-  selectionMode: 'multiple'
+  selectionMode: 'multiple',
+  decorators: [
+    (Story, {viewMode}) => <Story viewMode={viewMode} />
```

Comment:
Can you add this in the ActionBar example with CardView too? Looks like it's horizontally scrolling at the moment

Reply:
I've been poking around locally, the viewMode doesn't seem to have been working for quite some time actually, regardless if you run the storybook in docs mode or not. Still bisecting

Reply:
Apparently something from https://github.com/adobe/react-spectrum/pull/7435? Thats when viewMode becomes undefined for me locally but nothing in the PR jumps out to me

Reply:
Reverted the story fix since Rob has handled them in https://github.com/adobe/react-spectrum/pull/8272